test/feature/step_definitions/setup_definitions.coffee

```diff
@@ -34,3 +34,7 @@ module.exports = ->
     @createFile "routes.coffee", routesFileContents, (err) =>
       return done.fail err if err
       done()
+
+
+  @Then /^the app doesn't error$/, (done) ->
```

Comment:
Yeah it doesn't actually do anything, but does it read OK to have a scenario with only 2 Givens and no Thens?

```cucumber
Given a file "controllers/my_controller_spec.coffee" with the content
     """
     # This file doesn't export a controller
     """
And an exprestive app using defaults
```
vs
```cucumber
Given a file "controllers/my_controller_spec.coffee" with the content
     """
     # This file doesn't export a controller
     """
And an exprestive app using defaults
Then the app doesn't error
```